rye/src/config.rs

```diff
@@ -247,16 +247,16 @@ impl Config {
         Ok(rv)
     }
 
-    /// Indicates if the experimental puffin support should be used.
-    pub fn use_puffin(&self) -> bool {
+    /// Indicates if the experimental uv support should be used.
+    pub fn use_uf(&self) -> bool {
```

Comment:
I know this got merged already but this has a typo, it should probably be `use_uv` not `use_uf`.